fastlane/Fastfile

```diff
@@ -73,6 +73,7 @@ platform :ios do
         scheme: scheme
       ))
       increment_build_number_in_xcodeproj(
+          build_number: ENV["BUILD_NUMBER"],
```

Comment:
Can you point me to the documentation for this? Also should this be added between line 71 and 72?